src/include/sof/math/crossover_common.h

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Baofeng Tian <baofeng.tian@intel.com>
```

Comment:
pls also preserve the original author as this is this will be mostly copy and paste.

Reply:
I hope you meant "cut and paste". It would be sad for a refactoring PR to add copy/paste :-)

Reply:
sorry, I made mistake here, updating.